src/test/resources/project/untitled/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
So, its basically your intention to shit a mini-project for testing?

Reply:
Yes, it would make testing a lot easier. Currently, we have no way of running `mvn compile` or `gradle build` automatically, so I have provided a binary file of the class. In the future, once we have that covered, I will update the tests to generate the binary classes automatically.